application/fifo_dlq_handler/fifo_dlq_handler.py

```diff
@@ -4,6 +4,7 @@
 from common.dynamodb import add_change_request_to_dynamodb
 from common.middlewares import set_correlation_id, unhandled_exception_logging
 from common.utilities import extract_body, get_sequence_number
+from eventbridge_dlq_handler.eventbridge_dlq_handler import handle_sqs_msg_attributes
```

Comment:
could this move to common rather than having one dlq reference another

Reply:
I tried in the previous commit but I struggled with sharing the fixture @dead_letter_message.

Reply:
Had a look over the PR I think this is still an issue the `handle_sqs_msg_attributes` function is define in `common.utilities` but it being imported via the eventbridge dlq handle